### PR TITLE
Docs: style guide - colors & opacity

### PR DESCRIPTION
**Pull request by Illia-Sakharau**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [x] 📝 Documentation Update

## Description
- [x] add information about colors and opacity tokens to `styleguide-design.md`
- [x] rename tokens according to documentation

## Related Tickets & Documents

- Related Issue #46

## Screenshots, Recordings

Screenshot 2024-05-14 at 17 19 53

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

File: readme/styleguide-design.md [hunk 1]
```diff
@@ -1 +1,80 @@
...
+Please follow the rules and tokens provided in the guide.
+
+#### General rules
+1. Use kebab case notation for tokens naming.
+
+## 🎨 Colors
+### Usage
+```scss
+{
+  background-color: rgba($color-secondary-yellow, $opacity-8);
```
Comment (ansivgit) [inline readme/styleguide-design.md:13]:
```suggestion
  background-color: rgba($secondary-yellow, $opacity-8);
```

File: readme/styleguide-design.md [hunk 2]
```diff
@@ -1 +1,80 @@
...
+  color: $color-primary-black;
```
Comment (ansivgit) [inline readme/styleguide-design.md:14]: It seems to me, does not comply with the principle DRY

File: src/app/components/actions/actions.scss [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
   font-size: 18px;
   line-height: 28px;
-  color: $color-secondary-gray600;
+  color: $color-secondary-gray-600;
```
Comment (ansivgit) [inline src/app/components/actions/actions.scss:12]: Do we really need this changes?

File: src/app/components/principle-card/principle-card.scss [hunk 4]
```diff
@@ -107,7 +107,7 @@
     width: 20px;
     height: 19px;
 
-    background-color: $color-accent-yellow;
+    background-color: rgba($color-secondary-yellow, $opacity-80);
```
Comment (ansivgit) [inline src/app/components/principle-card/principle-card.scss:110]: I think it's definitely an accent color.

File: src/styles/_constants.scss [hunk 5]
```diff
@@ -14,20 +14,34 @@ $font-weight-regular: 400;
...
-$color-secondary-yellow-50: #fffaf0;
+
+// COLORS
+
+// Primary colors
+$color-primary-black: hsl(0deg 0% 0%);
+$color-primary-white: hsl(0deg 0% 100%);
+
+// Yellow colors
+$color-secondary-yellow: hsl(50deg 100% 56%);
```
Comment (ansivgit) [inline src/styles/_constants.scss:25]: Maybe we still need old color-format for refactoring? E.g:

```suggestion
$color-secondary-yellow: hsl(50deg 100% 56%); // #ffdb20
```

File: src/styles/_constants.scss [hunk 6]
```diff
@@ -14,20 +14,34 @@ $font-weight-regular: 400;
...
+// Secondary font colors
+$color-secondary-gray-600: hsl(0deg 0% 33%);
+$color-secondary-gray-500: hsl(0deg 0% 46%);
+$color-secondary-gray-400: hsl(0deg 0% 69%);
+$color-secondary-gray-200: hsl(0deg 0% 90%);
+$color-secondary-gray-100: hsl(0deg 0% 95%);
+
+
+// OPACITY
+$opacity-100: 1;
```
Comment (ansivgit) [inline src/styles/_constants.scss:42]: Do we really need this changes?

**Review by ramzes-84 — APPROVED** [review]

**Comment by andron13** [comment]
This branch has conflicts that must be resolved
Use the [web editor](https://github.com/rolling-scopes/site/pull/205/conflicts) or the  to resolve conflicts.
Conflicting files
src/app/components/social-media/social-media.scss

**Review by ansivgit — APPROVED** [review]

**Pull request merged**